config/config.go

```diff
@@ -0,0 +1,93 @@
+// Copyright © 2022 Meroxa, Inc.
```

Comment:
im not sure if this is fixed in the other PR, but just a note to make sure all the headers have 2024 instead
if the file is old and was created before this PR, then just leave it at 2022
```suggestion
// Copyright © 2024 Meroxa, Inc.
```